Updated no-jquery post for Ember 3.0.0

0xadada opened the pull request:
Ember 3.0.0 handles dispatched events natively, so the native event
dispatcher addon is only required for Ember 1 and 2.

Miguel, i'm implementing the changes in your article over here:
https://github.com/mirai-audio/mir/issues/31, and subscribed to your
remove-jquery PR over at the Torii addon here
https://github.com/Vestorly/torii/pull/399.

cheers 🍷,
-ron

imathis committed:
added site-wide description to config.yml (this provides a default meta description for the site, but can be overridden on on any page with page.description in the yaml)

imathis committed:
now Github repo url can be specified inline setup_github_pages command

hSATAC committed:
Fix delicious widget js error
JS error occured when there's no tags on bookmarks
Check if it's an empty string before join item[i].t

imathis committed:
Merge pull request #335 from hSATAC/master

Fix delicious widget js error

grahamc committed:
tilecase -> titlecase

imathis committed:
Merge pull request #351 from grahamc/master

Resolve a simple _config.yml typo

imathis committed:
Fix for annoying overflow scroll issues caused by facebook and twitter share/follow widgets.

imathis committed:
Merge branch 'master' of github.com:imathis/octopress

robertkowalski committed:
Fix CRLF, closes #374

dtchepak committed:
Extract category_link filter from category_generator.rb plugin

Allows category_link to be used as a filter to display a link to
any category.

joliss committed:
Fix typo

Commit:
Github -> GitHub (capitalization)

imathis committed:
Merge pull request #434 from nomulous/patch-1

Github -> GitHub (capitalization)

imathis committed:
Merge pull request #427 from joliss/typo

Fix typo

Commit:
Raise RuntimeError when gist is unavailable

Commit:
Convert to integer before comparing. data.code is String

Commit:
Fix for bottom margin in nested lists.

treyhunner committed:
Add EditorConfig file for Ruby files

treyhunner committed:
Add EditorConfig file for classic theme

imathis committed:
Merge pull request #376 from robertkowalski/clrf

Fix CRLF, closes #374

lukekarrys committed:
Don't display comments link for post excerpts on index page when comments for that post are set to false.

joseprio committed:
Added robots.txt, with sitemap link

acusti committed:
Fixed “3rd party plugins” wiki page link

Signed-off-by: Brandon Mathis <brandon@imathis.com>

imathis committed:
updated to latest version of Sass, fixed deprecation warnings

imathis committed:
Octopress filters are now limited to html producing templating languages, Fixes #536

imathis committed:
Merge pull request #499 from lukekarrys/patch-1

Don't display comments link for post excerpts on index page when comments for that post are set to false

imathis committed:
Updated Gemfile

imathis committed:
updated plugins to work with latest version of Liquid

omasanori committed:
Do not strip trailing slash of canonical URLs.

Usually, links for posts are generated with a trailing slash, but
canonical URLs aren't.  Some services (e.g. http://b.hatena.ne.jp/)
are confused with that behavior.

We can fix it by removing canonical URLs, but then anyone may read
a post with /index.html URL and post the URL to some tool or service.
Therefore rel="canonical" seems useful IMHO.

Signed-off-by: OGINO Masanori <masanori.ogino@gmail.com>

imathis committed:
Merge pull request #592 from omasanori/fix-rel-canonical

Do not strip trailing slash of canonical URLs.

zanshin committed:
Adding .powrc to allow RVM to work when POW is updated to version 0.4.0 and beyond.

Support for automatically handling .rvmrc files in projects is being deprecated in
POW version 0.4.0 and beyond. RVM will still work but project will need to have a
.powrc file that detects the presence of and then loads RVM.

The code in the .powrc could also be in the .powenv file, however the .powrc loads
first and is the configuration file recommended for inclusion in version control.

imathis committed:
Merge pull request #602 from zan5hin/powrc

Adding .powrc to allow RVM to work when POW is updated to version 0.4.0 ...

vyper committed:
Added stringex and changed to use #to_url for create directory. To don't receive error in url to accented characters.

earaya committed:
optimized order of styles and scripts in header

nathanl committed:
Example repo URL - see Github issue 516

zlu committed:
Updated Github API to V3.

Github API V2 has been removed.  The response data format also changed.

zlu committed:
Updated Github API to V3.

Github API V2 has been removed.  The response data format also changed.

zlu committed:
Updated Github API to V3.

Github API V2 has been removed.  The response data format also changed.

zlu committed:
Updated Github API to V3.

Github API V2 has been removed.  The response data format also changed.

imathis committed:
Merge pull request #666 from nathanl/example_repo_url

Example repo URL - see Github issue 516

imathis committed:
Updated gems and rubies to 1.9.3

imathis committed:
Merge branch 'master' of github.com:imathis/octopress

Confusion committed:
Fix for issue #622

The filters bundled with Octopress are not applied to files ending in 
`.md`, because the match in octopress_filters.rb does not match `.md`.

imathis committed:
Merge pull request #706 from Confusion/patch-1

Fix for issue #622

imathis committed:
bumped rbenv version fixing #743

imathis committed:
Merge branch 'master' of github.com:imathis/octopress

imathis committed:
Merge pull request #675 from zlu/master

Update Github API to V3

imathis committed:
Twitter API call and links are now protocol relative

imathis committed:
Merge branch 'master' of github.com:imathis/octopress

imathis committed:
Using https instead of protocol relative urls, for Pinboard and Twitter feeds

vaz committed:
setup_github_pages: allow for clones missing .git extension

imathis committed:
Disabled Rsync delete by default. Closes #787

vyper committed:
Upgrade ruby version

1e0ng committed:
To support url without domain name like /path/file

You need to know if you would like to support this, each word in the title can not start with "/".

parkr committed:
Fixing English error pet-peeve.

imathis committed:
Merge pull request #831 from parkr/english

Fixing English error pet-peeve.

jsvensson committed:
Rakefile option for any additional rsync arguments

Added variable `rsync_args`

imathis committed:
gist tag plugin now works with the latest changes GitHub gists and does not fail if you do not specify a filename.

imathis committed:
Merge pull request #824 from liangsun/patch-1

To support url without domain name like /path/file

imathis committed:
Merge pull request #844 from jsvensson/rsyncargs

Rakefile option for any additional rsync arguments

imathis committed:
Titles in code plugins can have slashes in them. Closes #892

imathis committed:
Removed unnecessary regex in code block plugin

imathis committed:
Merge pull request #408 from dtchepak/category_link_filter

Extract category_link filter from category_generator.rb plugin

imathis committed:
updated gems

imathis committed:
Merge pull request #537 from joseprio/master

robots.txt generation

imathis committed:
Rake task new_post now accepts titles through stdin
This should help resolve issues for zsh users and make it easier to enter titles which freak out Rake's argument processing

imathis committed:
removed accidentally added file

rixth committed:
Escape github repo descriptions, as they may contain HTML.

imathis committed:
Merge pull request #914 from rixth/master

Escape github repo descriptions, as they may contain HTML.

imathis committed:
updated ender.js to latest

seankerr committed:
Fixed blockquote plugin (author alone would not show up without title)

richo committed:
Whitespace

parkr committed:
Merge pull request #936 from richo/cleanup/whitespace

Whitespace

imathis committed:
Improved error message for category_generator plugin. Addresses #116

imathis committed:
Striped whitespace from titles in blockquote plugin

imathis committed:
Video tag plugin now supports mp4,ogv,webm formats, closes #931

imathis committed:
Removed unnecessary passive group from octopress origin regex

imathis committed:
improved regexes for setup_github_pages task

parkr committed:
Added travis.yml so travis doesn't fail for every master branch PR and push

parkr committed:
Added travis stamp of approval to README.

sjml committed:
Update plugins/pygments_code.rb

Making it so the build doesn't fail if a bad lexer is passed to the code highlighting.

rretsiem committed:
Async load of Facebook JS

added async switch = TRUE to loader.
According to Google PageSpeed the Facebook JS was not loaded async which blocks rendering of the page as long as the JS isn't fully loaded.

sjml committed:
Update plugins/pygments_code.rb

Giving a more helpful error message instead of silently failing.

floydpink committed:
fixed issue #949 using @alexkirmse's patch

floydpink committed:
Fixed issue with mobile nav choosing the last option by default. Fixes #950.

schwern committed:
Use the Github API repo sorting.

The Github API can sort by pushed time, no need to do it manually.
http://developer.github.com/v3/repos/#list-user-repositories

This is both more efficient, Github does the sorting, but it also
fixes a bug.  Because the list of results are paginated, if you have
more than a page's worth of repositories it will only sort that page.
This results in a false view of what has been recently pushed.

For a good example, try my github account "schwern" with and
without this patch.

parkr committed:
Merge pull request #1031 from schwern/github_repo_sorting

Use the Github API repo sorting.

parkr committed:
Use HTML5 '&' character to connect query params in GitHub API call. #1031.

parkr committed:
It's 2013. Hi, MIT License.

vyper committed:
Merge branch 'master' of github.com:imathis/octopress

Conflicts:
	plugins/category_generator.rb

Commit:
Bump sinatra to 1.3.5

parkr committed:
Merge pull request #1074 from joneslee85/bump-sinatra

Bump sinatra to 1.3.5

parkr committed:
Merge pull request #438 from roman-yepishev/gist_error

Raise RuntimeError when gist is unavailable - #437

parkr committed:
Using data.code.to_i like a true Rubyist in plugins/gist_code.rb. Heh. #438

parkr committed:
Using data.code.to_i like a true Rubyist in plugins/gist_code.rb. Heh. #438

parkr committed:
Merge pull request #448 from NSError/list-fix

Fix for bottom margin in nested lists.

parkr committed:
Merge branch 'master' of github.com:imathis/octopress

* 'master' of github.com:imathis/octopress:
  Using data.code.to_i like a true Rubyist in plugins/gist_code.rb. Heh. #438
  Fix for bottom margin in nested lists.

parkr committed:
Merge pull request #452 from treyhunner/topic/editorconfig

Add EditorConfig files to define indentation styles

parkr committed:
Merge pull request #629 from vyper/master

Added stringex and changed to use #to_url for create directory.

parkr committed:
Merge pull request #664 from earaya/master

Styles & scripts optimization in the document head

parkr committed:
Merge pull request #990 from sjml/patch-1

Generate can fail if Pygments can't parse a code block

parkr committed:
Merge pull request #991 from rretsiem/patch-2

Async load of Facebook JS

parkr committed:
Added .gitattributes file to keep CRLF's out of the codebase.

parkr committed:
Merge branch 'master' of github.com:imathis/octopress

* 'master' of github.com:imathis/octopress:
  Update plugins/pygments_code.rb
  Async load of Facebook JS
  Update plugins/pygments_code.rb
  Upgrade ruby version
  optimized order of styles and scripts in header
  Added stringex and changed to use #to_url for create directory. To don't receive error in url to accented characters.

imathis committed:
Removed Twitter and Ender.js. Added jQuery and updated Modernizr

peterboni committed:
Mobile nav fixed.

Now shows links instead of 'undefined'.

imathis committed:
removed Twitter configuration for timeline and follow buton which are no longer integrated

imathis committed:
Fixed GitHub jsonp call. Closed #1118

imathis committed:
Merge pull request #1116 from peterboni/patch-1

Mobile nav fixed.

imathis committed:
added config_tag plugin for integration of configuration into templates

imathis committed:
Improved: config_tag plugin is much more flexible now and can be used by other plugins directly through the config_tag method

imathis committed:
Added sass-globbing, allowing plugin stylesheets to be auto-imported from the assets/stylesheets/plugins directory

imathis committed:
fixed addSidebarToggler where index and section were flip-flopped

imathis committed:
fixed mobile navigation

imathis committed:
fixed video wrapping js

imathis committed:
Allowing dashes in code names. Closes #1166

imathis committed:
Accepting all non-whitespace characters as language names for Pygments code highlighting

imathis committed:
Added support for deploying to github.io

imathis committed:
Sinatra now correctly returns code 404 when a page is not found. Closes #1198

Commit:
Pick articles from local backup

Commit:
Removed marks from codeblocks

Commit:
Small fixed in post

Commit:
More fixes in posts

Commit:
Include forks

Commit:
Removed keyword

Commit:
New article about git merge/pull with strategy

Commit:
Pubished new article about git merge options

Commit:
Started post about sublime text snippet to transform to snake case.

Commit:
Articulo de elixir y dynamo

Commit:
Added styles.

Commit:
Renamed article

Commit:
added social media buttons in configuration file

Commit:
G+ username as ID

Commit:
New post: Rubytapas downloader

Commit:
Fixed typos in the rubytapas post.

cibernox committed:
Only specify the main version of ruby, not the pachset

cibernox committed:
New article about sass

cibernox committed:
Publish sass article

cibernox committed:
Blog post about grep

cibernox committed:
Small gramatical corrections

cibernox committed:
Fix typo

cibernox committed:
Ruby benchmark article

cibernox committed:
Fix several typos

cibernox committed:
Fix even more typos

cibernox committed:
Infinite typos

cibernox committed:
Updated benchmark with more complex tests

cibernox committed:
Added emberJS first meetup slides

cibernox committed:
Fix typo

cibernox committed:
WIP talk 'Meet ember js'

cibernox committed:
Add slide about the router

cibernox committed:
More slides about ember basic concepts

cibernox committed:
More slides in the Meet Ember talk

cibernox committed:
Presentation almost complete

cibernox committed:
Fix image

cibernox committed:
Presentation finished. Still details to polish.

cibernox committed:
Improve hidden notes

cibernox committed:
Finished!

cibernox committed:
Change title

cibernox committed:
Finished emberjs presentation.

cibernox committed:
Update talks index

cibernox committed:
disable remote controll in presentation

cibernox committed:
Ember components + web post

cibernox committed:
Rename blog post

cibernox committed:
Small collection in WebP + components post.

cibernox committed:
Fix bug in code

cibernox committed:
Set lates ruby

cibernox committed:
Add article about using glyphicons in ember-cli

cibernox committed:
Publish article explicitly.

cibernox committed:
Update bio to London.

cibernox committed:
Fix some typos.

cibernox committed:
Add paragraph

cibernox committed:
Added post about ember-cli

cibernox committed:
Fix some typos

cibernox committed:
Correct another typo

cibernox committed:
Use geround instead of infinitive

cibernox committed:
specify -> specific

cibernox committed:
Add missing verb.

cibernox committed:
Fix small typo

cibernox committed:
Update bio a bit.

cibernox committed:
New visual style

cibernox committed:
Deploy to GH pages working again

cibernox committed:
Reduce talks preview size

cibernox committed:
Change post url

cibernox committed:
Disable ajax form for now

cibernox committed:
Fix contact form

cibernox committed:
Added CNAME file in source dir

cibernox committed:
My name links to the about page

cibernox committed:
Improve visualization in mobile

cibernox committed:
Fix weird paragraphs in mobile

cibernox committed:
Improved talks section in mobile

cibernox committed:
Fix css bug with article separator

cibernox committed:
Update theme submodule without blur effect

cibernox committed:
Update ruby version

cibernox committed:
Post with screencast about ember.js

cibernox committed:
Add link  with repo

cibernox committed:
Add analytics again :S

cibernox committed:
Update ember-cli article with new 0.1.5 install command

cibernox committed:
Blog post about tampering responses with service workers

cibernox committed:
Added latest talk

cibernox committed:
Improve article

cibernox committed:
Use latest ruby

cibernox committed:
Update to latest rake

cibernox committed:
Added boubon and neat.

I don't know what changes but I need them to generate the blog.

cibernox committed:
Add blog post about EmbeddedRecordsMixin

cibernox committed:
Fix a typo

cibernox committed:
Add some refinements in article

cibernox committed:
Change a bit the end of the article

cibernox committed:
Change ruff / tough

cibernox committed:
Improve second last paragraph.

cibernox committed:
Fix two more typos

cibernox committed:
Fixed another typo

cibernox committed:
Blog post about the initializer stuff

cibernox committed:
Publish the post

cibernox committed:
Change blockquote

cibernox committed:
Mention nikz

cibernox committed:
Fix typos

cibernox committed:
Fix more typos

cibernox committed:
Fix 3 occurences it's then I meant to use a possesive

cibernox committed:
Fixed A LOT of typos

cibernox committed:
Added article about closure actions

cibernox committed:
Fix some wording

cibernox committed:
Corrected some wording

cibernox committed:
Final wording?

cibernox committed:
Probably last correction from my own proofreading

cibernox committed:
Fix another typo

cibernox committed:
lets <-> let's

cibernox committed:
Add breathing comma

cibernox committed:
Correct some things thx to Aaron

cibernox committed:
Fix more typos

cibernox committed:
Fix even more typos

cibernox committed:
Added note about extracting function from the root of your component

cibernox committed:
Last corrections

cibernox committed:
Added article with one trick

cibernox committed:
Fix some wording

cibernox committed:
Fix typo

cibernox committed:
Fix some typos and improve wording

cibernox committed:
Opps, bad link

cibernox committed:
Save stuff and fix outdated article

cibernox committed:
Fix header styles

cibernox committed:
Fix broken link in ancient blog post

cibernox committed:
Remove `this.attrs`

Is a bad practice

cibernox committed:
Add new article about better testing and removing jQuery

cibernox committed:
Fix style maybe?

cibernox committed:
Fix link

cibernox committed:
Some fixes

cibernox committed:
More fixes

cibernox committed:
More changes

cibernox committed:
Fixed more feedback

cibernox committed:
Fix typos

cibernox commented:
Thanks for the PR, but you made the changes in the generated HTML. That html is generated automatically from markdown files in the `source` branch. Could you make the PR changing the source files?

Also, I think that `ember-native-dom-event-dispatcher` doesn't work in Ember 1, only Ember 2.

0xadada committed:
Updated no-jquery post for Ember 3.0.0

Ember 3.0.0 handles dispatched events natively, so the native event dispatcher addon is only required for Ember 1 and 2.

Miguel, i'm implementing the changes in your article over here: https://github.com/mirai-audio/mir/issues/31, and subscribed to your remove-jquery PR over at the Torii addon here https://github.com/Vestorly/torii/pull/399.

cheers 🍷,
-ron
